common/djangoapps/student/models.py

```diff
@@ -1195,6 +1195,13 @@ def is_enrolled(cls, user, course_key):
         """
         if not user.is_authenticated():
             return False
+
+        # unwrap CCXLocators so that we use the course as the access control
+        # source
+        from ccx_keys.locator import CCXLocator
+        if isinstance(course_key, CCXLocator):
+            course_key = course_key.to_course_locator()
+
```

Comment:
@ormsbee just mentioned to me that he had talked to @pdpinch about this, and that they were hoping to actually use the serialized CCXCourseKeys in the enrollment object.

Also, I think you actually want this to be `CCXCourseKey`, not `CCXLocator` (since `CCXLocator` is a `UsageKey`, not a `CourseKey`)